src/TraceEvent/Parsers/KernelTraceEventParser.cs

```diff
@@ -7204,7 +7269,7 @@ protected internal override void Dispatch()
         {
             Debug.Assert(!(Version == 0 && EventDataLength != 20));
             Debug.Assert(!(Version == 1 && EventDataLength < HostOffset(28, 1)));   // TODO fixed by hand
-            Debug.Assert(!(Version > 1 && EventDataLength < HostOffset(28, 1)));
+            //Debug.Assert(!(Version > 1 && EventDataLength < HostOffset(28, 1)));
```

Comment:
Enable this. I do not see it triggered.

Reply:
Most other commented asserts do not seem triggered either.

---
In reply to: [384846815](https://github.com/microsoft/perfview/pull/1107#discussion_r384846815) [](ancestors = 384846815)